client-flink/common/src/main/java/org/apache/celeborn/plugin/flink/config/PluginConf.java

```diff
@@ -28,7 +28,11 @@ public enum PluginConf {
       "remote-shuffle.job.concurrent-readings-per-gate", "", String.valueOf(Integer.MAX_VALUE)),
   MEMORY_PER_RESULT_PARTITION("remote-shuffle.job.memory-per-partition", "", "64m"),
   MEMORY_PER_INPUT_GATE("remote-shuffle.job.memory-per-gate", "", "32m"),
+  SUPPORT_FLOATING_BUFFER_PER_INPUT_GATE(
+      "remote-shuffle.job.support-floating-buffer-per-input-gate", "", "false"),
```

Comment:
The default value need to be true. IMO, PluginConf need migrate to CelebornConf, and may use celeborn.client.flink as prefix. This can be modified in later pr.